raiden/tests/integration/test_regression.py

```diff
@@ -274,23 +277,36 @@ def run_regression_payment_complete_after_refund_to_the_initiator(
     # Topology:
     #
     #  0 -> 1 -> 2
-    #  v         ^
+    #  |         ^
+    #  v         |
     #  3 ------> 4
 
     app_channels = [(app0, app1), (app1, app2), (app0, app3), (app3, app4), (app4, app2)]
-
     open_and_wait_for_channels(app_channels, registry_address, token, deposit, settle_timeout)
 
+    transfer_timeout = 10
     # Use all deposit from app1->app2 to force a refund
     transfer(
-        initiator_app=app1, target_app=app2, token_address=token, amount=deposit, identifier=1
+        initiator_app=app1,
+        target_app=app2,
+        token_address=token,
+        amount=deposit,
+        identifier=PaymentID(1),
     )
 
     # Send a transfer that will result in a refund app1->app0
     transfer(
-        initiator_app=app0, target_app=app2, token_address=token, amount=deposit, identifier=2
+        initiator_app=app0,
+        target_app=app2,
+        token_address=token,
+        amount=deposit,
+        identifier=PaymentID(2),
+        timeout=transfer_timeout * 2,
```

Comment:
since you discovered the timeout for the second test is too slow and you made this new timeout variable why not  just do `transfer_timeout = 20` ? Is the value of `10` used anywhere else?

Reply:
10s is used everywhere, so I wanted to make clear that this is double, but you"re right. I can just put it in there.